src/librustdoc/messages.ftl

```diff
@@ -0,0 +1,7 @@
+rustdoc_main_error = {$error}
+
+rustdoc_couldnt_generate_documentation =
+    couldn't generate documentation: {$error}
+    .note = failed to create or modify "{$file}"
+
+rustdoc_compilation_failed = Compilation failed, aborting rustdoc
```

Comment:
I know this is pre-existing, but it's conventional for diagnostics to be lowercase. Not sure if rustdoc is special in this regard or if this was just an oversight.

```suggestion
rustdoc_compilation_failed = compilation failed, aborting rustdoc
```

Reply:
Most of the diagnostics in rustdoc are in lowercase (around < 5 that are not).

Shall we set all them to lowercase? @jsha